### PR TITLE
BEF-35 Implemented bulk update via multi_query

### PR DESCRIPTION
**Pull request by DrParanoia**

### CODE REVIEW

**Comment by kibertoad** [comment]
Test is missing.

**Comment by DrParanoia** [comment]
@kibertoad Added a simple test for multi-update functionality. The test environment, as well as the assertion methods, are really funky/questionable though 😆 Took a while to figure them out...

**Review by kibertoad — APPROVED** [review]

File: db.class.php [hunk 1]
```diff
@@ -597,6 +618,15 @@ public function queryRawUnbuf() { $args = func_get_args(); return $this->prepend
   protected function queryHelper() {
     $args = func_get_args();
     $type = array_shift($args);
+
+    $isMultiUpdate = false;
+
+    if (!empty($args) && $args[1] === 'multiUpdate') {
```
Comment (artursvonda) [inline db.class.php:624]: @DrParanoia Found a bug. This is causing notices when the count($args) === 1. Undefined index.

Comment (DrParanoia) [reply db.class.php:624]: @artursvonda is it critical? 😆 Or can we leave it when we next time "patch" MeekroDB?

**Pull request merged**